apps/design-system/src/subjects/views/connectors/connectors.tsx

```diff
@@ -53,7 +53,6 @@ const ConnectorsListPageContent = (): JSX.Element => {
           <ListActions.Root>
             <ListActions.Right>
               <Button
-                variant="default"
```

Comment:
Removed `default` variant and some other variants. If no variant is specified, then `solid` is the default variant and `primary` is the default `theme`